README.md

```diff
@@ -42,6 +42,8 @@ Where a field definition object takes the following form:
   fieldClassname: String data (optional) representing a custom CSS class for this field's form element,
   labelClassname: String data (optional) representing a custom CSS class for this field's label element,
   placeholder: String data (optional)
+  value: default data for this field (optional) String when "type" is checkbox, text, or textarea. Array of strings when "type" is checkboxGroup, choiceGroup, or multiplicity field.,
```

Comment:
I'd recommend calling this defaultValue if the image path is also prefixed with default-